Sources/FoundationEssentials/ProcessInfo/ProcessInfo.swift

```diff
@@ -466,12 +466,11 @@ extension _ProcessInfo {
 #if os(Linux)
     // Support for CFS quotas for cpu count as used by Docker.
     // Based on swift-nio code, https://github.com/apple/swift-nio/pull/1518
-    private static let cfsQuotaPath = "/sys/fs/cgroup/cpu/cpu.cfs_quota_us"
-    private static let cfsPeriodPath = "/sys/fs/cgroup/cpu/cpu.cfs_period_us"
-    private static let cpuSetPath = "/sys/fs/cgroup/cpuset/cpuset.cpus"
+    private static let cfsQuotaPath = URL(filePath: "/sys/fs/cgroup/cpu/cpu.cfs_quota_us", directoryHint: .notDirectory)
+    private static let cfsPeriodPath = URL(filePath: "/sys/fs/cgroup/cpu/cpu.cfs_period_us", directoryHint: .notDirectory)
+    private static let cpuSetPath = URL(filePath: "/sys/fs/cgroup/cpuset/cpuset.cpus", directoryHint: .notDirectory)
 
-    private static func firstLineOfFile(path: String) throws -> Substring {
-        // TODO: Replace with URL version once that is available in FoundationEssentials
+    private static func firstLineOfFile(path: URL) throws -> Substring {
```

Comment:
nit: I think we prefer calling these `URL`-type arguments and variables something like `url` rather than `path`. I understand that's probably going to clutter this diff though so I'm ok with leaving them as-is for now

Reply:
That's a fair point, I went ahead and updated it